src/index.ts

```diff
@@ -68,6 +68,10 @@ export interface ServerlessClamscanProps {
    * An optional list of S3 buckets to configure for ClamAV Virus Scanning; buckets can be added later by calling addSourceBucket.
    */
   readonly buckets?: Bucket[];
+  /**
+   * Reserved capacity of the scanning Lambda to throttle the rate of requests and do not use overall AWS Region capacity.
+   */
+  readonly reservedCapacity?: number;
```

Comment:
```suggestion
  readonly reservedConcurrency?: number;
```

Updating to match the feature name